pkg/generators/forwarding/fluentd/syslog_conf.go

```diff
@@ -72,6 +72,18 @@ func (conf *outputLabelConf) Tag() string {
 	return tag
 }
 
+func (conf *outputLabelConf) SyslogProtocol() string {
+	switch strings.ToLower(conf.Protocol()) {
+	case "tls":
+		return "tcp"
+	case "":
+		return "tcp"
+	case "udp":
+		return "udp"
+	}
```

Comment:
nit. Consider using constants from golang for protocol names.

Reply:
i didnt find which defines tcp/udp/tls all in same